test/libsolidity/syntaxTests/nameAndTypeResolution/448_no_unused_dec_after_use.sol

```diff
@@ -4,3 +4,5 @@ contract C {
         uint a;
     }
 }
+// ----
+// DeclarationError: (52-53): Undeclared identifier. Did you mean "a"?
```

Comment:
``Did you mean "a"?`` is rather misleading here - maybe we should change or remove that suggestion.